lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,17 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_key, from: nil, till: nil, outcome: nil, key: nil, primary: false)
```

Comment:
De `from` en `till` slaan op de begin- en einddatum van dit zorgdomein, zoals bepaald door het SBG.
De `sbg_key` is de code voor het zorgdomein. De `key` is hier de sbg code voor een specifieke vragenlijst binnen het domein.

Ik heb het ook niet bedacht dat dit bijgehouden moet worden. De betekenis kun je vinden in de sbg codelijsten.

Het verschil tussen `sbg_key` en `key` is inderdaad misschien wat magertjes, how about: `sbg_code` en `sbg_key`.